File: A1/A1-033/A1-033.cpp
#include<iostream>
using namespace std;

int main()
{
    // This makes std::cin and std::cout faster
    ios_base::sync_with_stdio(false); cin.tie(0);
    
    // Vowels
    char VOWELS[5] = {'A', 'E', 'I', 'O', 'U'};
    // Input
    int n, cnt = 0;;
    cin >> n;
    for(int i = 0; i < n; i++)
    {
        char letter;
        cin >> letter;
        // Count Vowels
        for(int j = 0; j < 5; j++)
        {
            if(letter == VOWELS[j]) { cnt++; }
        }
    }
    // Output
    cout << cnt;
    return 0;
}

File: A1/A1-033/A1-033.py
# --------------------------------------------------
# File Name : A1-033.py
# Problem   : Count Vowels
# Author    : Worralop Srichainont
# Date      : 2026-03-13
# --------------------------------------------------

# Vowel constants
VOWELS = ("A", "E", "I", "O", "U")

# Input number of letters
n_letters = int(input())

# Count vowels
vowel_count = 0
for _ in range(n_letters):
    letter = input().strip()
    if letter in VOWELS:
        vowel_count += 1

# Output
print(vowel_count)
